Configuration/Geometry/README.md

```diff
@@ -5,9 +5,9 @@ Different versions of various subdetectors can be combined. The available versio
 Tracker:
 * T1: Phase2 tilted tracker (v2016-04-12) w/ phase 1 (extended) pixel - DEPRECATED, superseded by T3
 * T2: Phase2 flat tracker (v2016-04-12) w/ phase 1 (extended) pixel- DEPRECATED, superseded by T4
-* T3: Phase2 tilted tracker (v3.6.5) w/ phase 2 pixel (v4.0.2.6)
+* T3: Phase2 tilted tracker (v3.6.5) w/ phase 2 pixel (v4.0.2.6) - DEPRECATED, superseded by T5
 * T4: Phase2 flat tracker (v2016-04-12) w/ phase 2 pixel (v4.0.2.6)
-* T5: Phase2 tilted tracker (v3.6.13) w/ phase 2 pixel (v4.0.2.5)
+* T5: Phase2 tilted tracker (v3.6.13) w/ phase 2 pixel (v4.0.2.5) 
```

Comment:
Ah just a  pico thing : T5 corresponds to OT v6.1.3 in tkLayout.
T5: Phase2 tilted tracker (v6.1.3) w/ phase 2 pixel (v4.0.2.5)